opsdroid/__main__.py

```diff
@@ -137,6 +145,10 @@ def welcome_message(config):
 @click.option('--version', '-v', is_flag=True, callback=print_version,
               expose_value=False, default=False, is_eager=True,
               help='Print the version and exit.')
+@click.option('--edit-config', '-c', is_flag=True, callback=edit_config,
```

Comment:
I love this idea! Really really useful.

We should have a look at other command line applications which have this kind of functionality and see what flags they use as I'm not 100% happy with these.

We could go for `--edit-config/-e`, which would be familiar to people who have run `crontab -e`. Or we could go for something like `--configure/-c`.